test/e2e/usersignup_test.go

```diff
@@ -238,6 +242,76 @@ func (s *userSignupIntegrationTest) TestUserIDAndAccountIDClaimsPropagated() {
 	VerifyResourcesProvisionedForSignup(s.T(), s.Awaitilities, userSignup, "deactivate30", "base")
 }
 
+func (s *userSignupIntegrationTest) TestGetSignupEndpointUpdatesIdentityClaims() {
+	hostAwait := s.Host()
+
+	// given
+	hostAwait.UpdateToolchainConfig(s.T(), testconfig.AutomaticApproval().Enabled(true))
+
+	id, err := uuid.NewV4()
+	require.NoError(s.T(), err)
+
+	// when
+	userSignup, _ := NewSignupRequest(s.Awaitilities).
+		Username("test-user-identityclaims").
+		Email("test-user-identityclaims@redhat.com").
+		IdentityID(id).
+		UserID("000999").
+		AccountID("111999").
+		EnsureMUR().
+		RequireConditions(wait.ConditionSet(wait.Default(), wait.ApprovedAutomatically())...).
+		Execute(s.T()).Resources()
+
+	// then
+	VerifyResourcesProvisionedForSignup(s.T(), s.Awaitilities, userSignup, "deactivate30", "base")
+
+	// Create a token and identity to invoke the GetSignup endpoint with
+	userIdentity := &commonauth.Identity{
+		ID:       id,
+		Username: userSignup.Spec.Username,
+	}
+	claims := []commonauth.ExtraClaim{commonauth.WithEmailClaim("test-user-updated@redhat.com")}
+	claims = append(claims, commonauth.WithOriginalSubClaim("updated-original-sub"))
+	claims = append(claims, commonauth.WithUserIDClaim("111222"))
+	claims = append(claims, commonauth.WithAccountIDClaim("999111"))
+	claims = append(claims, commonauth.WithPreferredUsernameClaim("test-user-updated"))
+	claims = append(claims, commonauth.WithGivenNameClaim("Jane"))
+	claims = append(claims, commonauth.WithFamilyNameClaim("Turner"))
+	claims = append(claims, commonauth.WithCompanyClaim("Acme"))
+
+	token, err := authsupport.NewTokenFromIdentity(userIdentity, claims...)
+	require.NoError(s.T(), err)
+
+	// Call the GET signup endpoint - we don't care about the response, we just want it to update the UserSignup
+	req, err := http.NewRequest("GET", hostAwait.RegistrationServiceURL+"/api/v1/signup", nil)
+	require.NoError(s.T(), err)
+	req.Header.Set("Authorization", "Bearer "+token)
+	req.Header.Set("content-type", "application/json")
+
+	req.Close = true
+	resp, err := httpClient.Do(req) // nolint:bodyclose // see `defer Close(t, resp)`
+	require.NoError(s.T(), err)
+	b, err := io.ReadAll(resp.Body)
+	require.Equal(s.T(), http.StatusOK, resp.StatusCode, "unexpected status, body: %s", string(b))
+
+	defer Close(s.T(), resp)
```

Comment:
minor thing - but probably you could use `InvokeEnpoint` utility here. 
See: https://github.com/codeready-toolchain/toolchain-e2e/blob/8eeaeaf8f0c07e64a65df8dbba596fecfb28dd1f/testsupport/regsvc.go#L14

Reply:
Great suggestion, thanks Francisc